pkg/plugin/vector/service.go

```diff
@@ -13,6 +13,7 @@ import (
 
 type Service interface {
 	Search(ctx context.Context, collection string, query string, topK uint64, filter map[string]interface{}) ([]store.SearchResult, error)
+	Health(ctx context.Context) (bool, error)
```

Comment:
Can this just be `Health(ctx context.Context) error`? A failing health check with no error message (i.e. a `return false, nil`) isn't much use to users, we should discourage implementors from returning it